cmake/util.cmake

```diff
@@ -70,7 +70,7 @@ MACRO(COPY_FILE_TO_TARGET _target _file)
 	if(UNIX)
 		ADD_CUSTOM_COMMAND(
 			TARGET ${_target} POST_BUILD
-			COMMAND cp -a "${_file}"  "$<TARGET_FILE_DIR:${_target}>/${LIBRAY_DESTINATION}/"
+			COMMAND cp -au "${_file}"  "$<TARGET_FILE_DIR:${_target}>/${LIBRAY_DESTINATION}/"
```

Comment:
this seems like an accidental change? It breaks compilation on MacOS....

Reply:
There was a build break in a previous iteration where a file was copied twice. I looked up the appropriate option in my `man cp` but of course Mac has to be special with their tools. I changed it to always use the CMake variant.

Reply:
Whenever I change something, another thing breaks, I'll just revert these changes...